### PR TITLE
changed html formatting, added website, email link for projects, adde…

### LINKED ISSUE #1: Make the project popups more direct
Button that takes them to info page
Button that takes them to slack channel
smaller logo

### PR DESCRIPTION
**Pull request by MihirDarji**
…d footer buttons for projects

resolve #1

### CODE REVIEW

**Comment by ZacharyRSmith** [comment]
it looks like a lot of the diff is due to white-space changes. Would you please add comments to the diff highlighting the non white-space changes?

**Comment by bld2104** [comment]
@MihirDarji when we merged the other pul requests it created a conflict with this one, sorry! Do you mind changing whatever is conflicting and re-sending?

**Comment by MihirDarji** [comment]
please try now, resolved conflicts

File: index.html [hunk 1]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
+                                    </svg>
+                                    <a href="http://www.phlask.me">www.phlask.me</a>
+                                    &nbsp;&nbsp;&nbsp;
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                                        class="bi bi-envelope" viewBox="0 0 16 16">
+                                        <path
+                                            d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm2-1a1 1 0 0 0-1 1v.217l7 4.2 7-4.2V4a1 1 0 0 0-1-1H2zm13 2.383-4.758 2.855L15 11.114v-5.73zm-.034 6.878L9.271 8.82 8 9.583 6.728 8.82l-5.694 3.44A1 1 0 0 0 2 13h12a1 1 0 0 0 .966-.739zM1 11.114l4.758-2.876L1 5.383v5.73z" />
+                                    </svg>
+                                    <a href="mailto:hello@phlask.me">hello@phlask.me</a>
+                                </p>
```
Comment (MihirDarji) [inline index.html:751]: justified text and added links for website and email

File: index.html [hunk 2]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
                             </div>
                         </div>
                     </div>
                 </div>
+                <div class="modal-footer">
+                    <a href="http://www.phlask.me" class="btn btn-primary">Learn More</a>
+                    <a href="http://www.phlask.me" class="btn btn-primary">Get Involved</a>
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                </div>
```
Comment (MihirDarji) [inline index.html:760]: added modal footer with buttons

File: index.html [hunk 3]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
                             </div>
+
                         </div>
                     </div>
                 </div>
+                <div class="modal-footer">
+                    <a href="https://codeforphilly.org/projects/paws_data_pipeline" class="btn btn-primary">Learn More</a>
+                    <a href="https://codeforphilly.org/projects/paws_data_pipeline" class="btn btn-primary">Get Involved</a>
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                </div>
```
Comment (MihirDarji) [inline index.html:836]: added modal footer with buttons

File: index.html [hunk 4]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
+                                
                             </div>
                         </div>
                     </div>
                 </div>
+                <div class="modal-footer">
+                    <a href="http://www.phlask.me" class="btn btn-primary">Learn More</a>
+                    <a href="http://www.phlask.me" class="btn btn-primary">Get Involved</a>
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                </div>
```
Comment (MihirDarji) [inline index.html:890]: added modal footer with buttons

File: index.html [hunk 5]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
                             </div>
                         </div>
                     </div>
                 </div>
+                <div class="modal-footer">
+                    <a href="https://codeforphilly.org/projects/philadelphia_lawyers_for_social_equity_-_record_expungement" class="btn btn-primary">Learn More</a>
+                    <a href="https://codeforphilly.org/projects/philadelphia_lawyers_for_social_equity_-_record_expungement" class="btn btn-primary">Get Involved</a>
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                </div>
             </div>
```
Comment (MihirDarji) [inline index.html:949]: added modal footer with buttons

File: index.html [hunk 6]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
                             </div>
                         </div>
                     </div>
                 </div>
+                <div class="modal-footer">
+                    <a href="https://codeforphilly.org/projects/philadelphia_bail_fund_interactive_data_portal" class="btn btn-primary">Learn More</a>
+                    <a href="https://codeforphilly.org/projects/philadelphia_bail_fund_interactive_data_portal" class="btn btn-primary">Get Involved</a>
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                </div>
             </div>
```
Comment (MihirDarji) [inline index.html:1008]: added modal footer with buttons

File: index.html [hunk 7]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
+                                </p>
                             </div>
                         </div>
                     </div>
                 </div>
+                <div class="modal-footer">
+                    <a href="http://www.phlask.me" class="btn btn-primary">Learn More</a>
+                    <a href="http://www.phlask.me" class="btn btn-primary">Get Involved</a>
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                </div>
```
Comment (MihirDarji) [inline index.html:1051]: added modal footer with buttons

File: index.html [hunk 8]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
+                                    </svg>
+                                    <a href="https://github.com/CodeForPhilly/cfp-mockup/">https://github.com/CodeForPhilly/cfp-mockup/</a>
+                                    &nbsp;&nbsp;&nbsp;
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                                        class="bi bi-envelope" viewBox="0 0 16 16">
+                                        <path
+                                            d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm2-1a1 1 0 0 0-1 1v.217l7 4.2 7-4.2V4a1 1 0 0 0-1-1H2zm13 2.383-4.758 2.855L15 11.114v-5.73zm-.034 6.878L9.271 8.82 8 9.583 6.728 8.82l-5.694 3.44A1 1 0 0 0 2 13h12a1 1 0 0 0 .966-.739zM1 11.114l4.758-2.876L1 5.383v5.73z" />
+                                    </svg>
+                                    <a href="mailto:email@goes.here">email@goes.here</a>
+                                </p>
```
Comment (MihirDarji) [inline index.html:1042]: justified text and added links for website and email

File: index.html [hunk 9]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
+                                    <a href="https://codeforphilly.org/projects/philadelphia_bail_fund_interactive_data_portal">codeforphilly.org/projects/philadelphia_bail_fund_interactive_data_portal</a>
+                                    &nbsp;&nbsp;&nbsp;
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                                        class="bi bi-envelope" viewBox="0 0 16 16">
+                                        <path
+                                            d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm2-1a1 1 0 0 0-1 1v.217l7 4.2 7-4.2V4a1 1 0 0 0-1-1H2zm13 2.383-4.758 2.855L15 11.114v-5.73zm-.034 6.878L9.271 8.82 8 9.583 6.728 8.82l-5.694 3.44A1 1 0 0 0 2 13h12a1 1 0 0 0 .966-.739zM1 11.114l4.758-2.876L1 5.383v5.73z" />
+                                    </svg>
+                                    <a href="mailto:email@goes.here">email@goes.here</a>
+                                </p>
                             </div>
```
Comment (MihirDarji) [inline index.html:999]: justified text and added links for website and email

File: index.html [hunk 10]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
+                                    &nbsp;&nbsp;&nbsp;
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                                        class="bi bi-envelope" viewBox="0 0 16 16">
+                                        <path
+                                            d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm2-1a1 1 0 0 0-1 1v.217l7 4.2 7-4.2V4a1 1 0 0 0-1-1H2zm13 2.383-4.758 2.855L15 11.114v-5.73zm-.034 6.878L9.271 8.82 8 9.583 6.728 8.82l-5.694 3.44A1 1 0 0 0 2 13h12a1 1 0 0 0 .966-.739zM1 11.114l4.758-2.876L1 5.383v5.73z" />
+                                    </svg>
+                                    <a href="mailto:hello@phlask.me">hello@phlask.me</a>
+                                </p>
+                                
                             </div>
```
Comment (MihirDarji) [inline index.html:940]: justified text and added links for website and email

File: index.html [hunk 11]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
+                                    &nbsp;&nbsp;&nbsp;
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                                        class="bi bi-envelope" viewBox="0 0 16 16">
+                                        <path
+                                            d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm2-1a1 1 0 0 0-1 1v.217l7 4.2 7-4.2V4a1 1 0 0 0-1-1H2zm13 2.383-4.758 2.855L15 11.114v-5.73zm-.034 6.878L9.271 8.82 8 9.583 6.728 8.82l-5.694 3.44A1 1 0 0 0 2 13h12a1 1 0 0 0 .966-.739zM1 11.114l4.758-2.876L1 5.383v5.73z" />
+                                    </svg>
+                                    <a href="mailto:email@goes.here">email@goes.here</a>
+                                </p>
+                                
                             </div>
```
Comment (MihirDarji) [inline index.html:882]: justified text and added links for website and email

File: index.html [hunk 12]
```diff
@@ -698,180 +714,341 @@ <h3 class="section-subheading text-muted">Come join us</h3>
...
+                                        <a href="https://codeforphilly.org/projects/paws_data_pipeline">codeforphilly.org/projects/paws_data_pipeline</a>
+                                        &nbsp;&nbsp;&nbsp;
+                                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                                            class="bi bi-envelope" viewBox="0 0 16 16">
+                                            <path
+                                                d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm2-1a1 1 0 0 0-1 1v.217l7 4.2 7-4.2V4a1 1 0 0 0-1-1H2zm13 2.383-4.758 2.855L15 11.114v-5.73zm-.034 6.878L9.271 8.82 8 9.583 6.728 8.82l-5.694 3.44A1 1 0 0 0 2 13h12a1 1 0 0 0 .966-.739zM1 11.114l4.758-2.876L1 5.383v5.73z" />
+                                        </svg>
+                                        <a href="mailto:email@email.here">email@email.here</a>
+                                    </p>
+
```
Comment (MihirDarji) [inline index.html:826]: justified text and added links for website and email

**Pull request merged**